### ISSUE TITLE
Integrations

### ISSUE THREAD

**Issue by willegan** [groupon/greenscreen#61]
Would be awesome to integrate into different services such as Google Analytics, Jira, Zendesk etc.

**Comment by ArashiDes** [comment]
Yeah is a good idea, but I want to get the different screen layout options done first. :)

**Comment by joshstrange** [comment]
@willegan I don't think a single Groupon employee has responded to or even looked at these issues in a long time. There are a couple forks of this that you might have more luck with, I'm not even sure if groupon is accepting pull requests.

**Comment by superhawk610** [comment]
Hope I'm not too late to the party - I just published [multicast](https://github.com/superhawk610/multicast), Greenscreen's spiritual successor for the new Castv2 API via mDNS. I believe Greenscreen was lead by an intern who has since left the company, so I don't imagine this repo will be getting (m)any more updates, but we're continuing active development over at multicast.

@willegan integrations sound like an excellent idea! And @alphasloth I have a guide for adding channel layouts on the [README](https://github.com/superhawk610/multicast) for multicast, we'd really appreciate the contribution if you get a chance.
